src/services/infrastructure/entity-resolver/timeline.resolver.service.ts

```diff
@@ -70,9 +68,11 @@ export class TimelineResolverService {
   ): Promise<string | undefined> {
     const result = await this.entityManager.findOne(Space, {
```

Comment:
This is dangerous. We can't imply that a calendarID is for a Space anymore. There should be, at the very least, defensive logic against it.